src/monopoly/view/monopoly_view.js

```diff
@@ -238,7 +238,51 @@ MonopolyView.prototype.drawCard = function(xPos, yPos, width, height, text, colo
     card.addChild(cardText);
 
     return card;
-}
+};
+
+
+
+MonopolyView.prototype.setupPropertyClick = function(tile, property){
+    tile.interactive = true;
+    var context = this;
+    //this sets the clickable area - doesnt work
```

Comment:
confusing comments (ie. does it work?) 

also there's a ton of commented code lying around that should all be removed unless there's an explicit reason we might want it around